test/Header/MessageIdTest.php

```diff
@@ -14,7 +14,7 @@
 
 /**
  * @group      Zend_Mail
- * @covers Zend\Mail\Header\MessageId<extended>
+ * @covers \Zend\Mail\Header\MessageId<extended>
```

Comment:
@asgrim no, @covers _doesn't_ need it, I use it in my tests without `\`  https://github.com/samsonasik/SanSessionToolbar/blob/master/test/Factory/Controller/SessionToolbarControllerFactoryTest.php#L81-L82 and it detect correctly